### DISCUSSION TITLE
Power Management / Screensaver: Clarify meaning and function

### DISCUSSION THREAD

**Discussion by shade-belisar** [linuxmint/discussions#676] [Ideas]
EDIT: This was originally an issue about functionality regarding sleep / lid closing behaviour. However, over the course of the ticket I was made aware that this functionality already exists. The way that the Power Management and Screensaver settings interact and work is however very confusing, at least to me. See my [comment](https://github.com/orgs/linuxmint/discussions/676#discussioncomment-11267370) further down for my issues.

When I close my laptop, I would like to keep it running but turn off the display (since I can't see it anyway). Currently I can select "Do nothing" and keep the machine running, but the display will also stay on.

This could be achieved by adding an option "Turn off screen" to the drop down menus for "When the lid is closed" in Power Management.

**Comment by claudiux** [comment]
In the meantime, you can use the Exit@claudiux applet, which offers the Screen Off option in its menu.

https://cinnamon-spices.linuxmint.com/applets/view/386

**Comment by shade-belisar** [comment]
Thanks for the suggestions, but it unfortunately does not really help. While I can turn the screen off and then close the lid, the screen turns back on as soon as I interact with the device (e.g. press play/pause on my headset).

**Comment by claudiux** [comment]
https://ubuntuhandbook.org/index.php/2020/05/lid-close-behavior-ubuntu-20-04/

https://forums.linuxmint.com/viewtopic.php?t=418957

**Comment by shade-belisar** [comment]
I'm afraid I don't understand your comment. Neither of those links describes the behavior I'm looking for. ignore leaves the screen on, lock requires me to log in again when opening the laptop and suspend sends it to sleep. I guess you could describe what I want as lock, but without having to enter the password when I open the laptop again.

**Comment by claudiux** [comment]
On my laptop, I've configured Power Management and Screensaver as this. When I close my laptop, only screen becomes off.

Power Management:
![PowerManagement](https://github.com/user-attachments/assets/78bb3b80-f897-4f4a-95d4-f90c11c4afe4)

Screensaver Settings tab:
![Screensaver1](https://github.com/user-attachments/assets/8a252970-b3d7-4de5-8cab-d216f1c0390d)

Screensaver Customize tab:
![Screensaver2](https://github.com/user-attachments/assets/493bb43d-9fbd-4880-91ed-241307c2d527)

**Comment by shade-belisar** [comment]
Wait, why does this work? I mean, thank you, that's what I was looking for. But this is in no way how I would have expected those settings to work. A quick list of the things I find confusing / misleading:

1. I would have expected "Lock Screen" to lead to the laptop being locked when the lid is closed. It seems to mean "Turn off screen".
2. I did not expect screensaver settings to be in any way relevant to this issue. As far as I know, a screensaver is a slowly moving image displayed on the screen after a period of inactivity. But as I just noticed, the description in the menu is "Manage screen lock settings". In my mind a screensaver and a lock screen are two different things.
3. In any case I do not understand why "Lock the computer after the screensaver starts" has anything to do with "Lock screen" when closing the lid. Neither screensaver nor lock screen start when I close the lid, it just goes black. Or is the screen going back said "screensaver"?

I'll rename the issue, because it turns out my actual issue is not the functionality, but the UX controlling it. I'm not even sure how this should be approached, but I would like to note that the current state, at least for me, is very confusing.

**Comment by claudiux** [comment]
$ `cinnamon-screensaver --help`

The screensaver manages the lock screen.
